### ISSUE TITLE
Remote ssh fail: unreachable or not Linux x86_64

### ISSUE THREAD

**Issue by thesby** [microsoft/vscode-remote-release#299]
Mac remote connection machine need to login  in two steps:
1. Login to jump machine
2. ssh connect to my machine from jump machine

But connection failed at the first step

**Comment by roblourens** [comment]
If you are using password auth, please set `remote.SSH.showLoginTerminal`

See https://code.visualstudio.com/docs/remote/troubleshooting#_enabling-alternate-ssh-authentication-methods for more details

**Comment by thesby** [comment]
Yes, `remote.SSH.showLoginTerminal` has already been set, so I can input my password.
When I input a false password, it reminds me a false password and requires me to input again. But when I input a correct password, this error message prompt and the terminal closed.

**Comment by roblourens** [comment]
Can you try again and share the log from the "Remote - SSH" output channel?

**Comment by rmorabia** [comment]
Having a similar issue, so going to post my output here: 

I have two servers. One requires a password. Terminal setting is enabled as well. When I try to connect to the password server, it asks me for my password (and will say my password is wrong if it is the wrong one), and when it's correct, I get the same error, but then it defaults to my other server which doesn't require a password.

Output is pretty simple:

```
Confirming that pwserver is a valid reachable host
Running type "C:\Users\hi\AppData\Local\Temp\vscode-linux-multi-line-command-pwserver.sh" | ssh  pwserver to confirm the host platform
ConfigFileWatcher: closing output:extension-output-%232
> 
> Pseudo-terminal will not be allocated because stdin is not a terminal.
> 
> 
> radhika@serverdomain's password:
> 
> 
> Permission denied, please try again.
> radhika@serverdomain's password:
> 
> 
> Welcome to Ubuntu 14.04.6 LTS (GNU/Linux 3.2.0-41-virtual i686)
> 
>  * Documentation:  https://help.ubuntu.com/
> New release '16.04.6 LTS' available.
> Run 'do-release-upgrade' to upgrade to it.
> 
> 
> Linux i686
> 
"uname" terminal command done
pwserver: unreachable or not Linux x86_64. (Linux i686)
```

The other Ubuntu server is the wrong/other server.

**Comment by roblourens** [comment]
> but then it defaults to my other server which doesn't require a password.

Sorry I'm not sure what you mean or where the "other" server comes in.

> Linux i686

We only support 64-bit linux servers, so this log just shows connecting to a 32-bit linux server, which is not supported.

**Comment by rmorabia** [comment]
Ah! That makes sense! Thank you for clearing that up for me.

**Comment by roblourens** [comment]
There is a feature request for 32 bit linux here: https://github.com/microsoft/vscode-remote-release/issues/67

**Comment by roblourens** [comment]
Oops leaving open for the OP

**Comment by fmscole** [comment]
My remote server is windows server  with opensshd.
The error msg:

Can't connect to lab: unreachable or not Linux x86_64 (C:\Program Files\OpenSSH\home\Administrator>More?)

I think it have connected to the remote server ,but some thing feiled.
what is "Administrator>More?)“  meaning?

**Comment by roblourens** [comment]
No windows servers are not supported.

**Comment by fmscole** [comment]
> 
> 
> No windows servers are not supported.

But what's wrong  with this?
```
Can't connect to lab: unreachable or not Linux x86_64 (C:\Program Files\OpenSSH\home\Administrator>More?)
```

**Comment by roblourens** [comment]
It is not a good error message but it basically correct, the machine is not Linux.

**Comment by ponoyoshi** [comment]
Hello, concerning this error, I have a similar problem without any informations, created a vm on ubuntu (32-bit) i686

installed ssh-server on it, then tried my ssh config by using cmd

Host VMSTRAUT
    HostName xxx.x.x.x
    User straut
    Port 22

connection seems to work fine using ssh VMSTRAUT
but when I use it in Visual studio, he ask me for password, I enter it then it return me an error:

![3CUv6VK](https://user-images.githubusercontent.com/22726871/58822993-fcdad080-8638-11e9-9c2e-3df4079edc29.png)

do I need to setup my VM in another way? seems to be the issue...
thanks!

**Comment by roblourens** [comment]
That is not a 64 bit Linux VM, we only support 64 bit Linux

**Comment by roblourens** [comment]
Have not heard back from OP, closing
